lib/requester/core.js

```diff
@@ -189,24 +210,27 @@ module.exports = {
         // Ensures that "request" creates URL encoded formdata or querystring as
         // foo=bar&foo=baz instead of foo[0]=bar&foo[1]=baz
         options.useQuerystring = true;
-        options.strictSSL = defaultOpts.strictSSL;
+        options.strictSSL = resolveWithProtocolProfileBehavior('strictSSL', defaultOpts, protocolProfileBehavior);
```

Comment:
Turn this into a map instead, so that the same statement does not have to be repeated over and over again.